src/3d/symbols/qgspointcloud3dsymbol.cpp

```diff
@@ -25,16 +25,22 @@
 // QgsPointCloud3DSymbol
 
 
-QgsPointCloud3DSymbol::QgsPointCloud3DSymbol()
+QgsPointCloud3DSymbol::QgsPointCloud3DSymbol( QgsPointCloud3DSymbol::RenderingStyle renderingStyle )
   : QgsAbstract3DSymbol()
+  , mRenderingStyle( renderingStyle )
```

Comment:
This would be better handled as a pure virtual method in QgsPointCloud3DSymbol, rather than passing it to the constructor